source/require-modules.txt

```diff
@@ -0,0 +1,107 @@
+.. _vsce-require:
+
+=============================================
+Use ``require()`` to Include External Modules
+=============================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/fact-vsce-preview.rst
+
+.. important::
+
+   A complete description of Node.js, modules, and the ``require()``
+   function is out of scope for this tutorial. To learn more, refer to
+   the `Node.js Documentation <https://nodejs.org/api/modules.html>`__.
+
+You can use the ``require()`` function in your MongoDB Playgrounds to
+include modules which exist in separate files.
+
+Require Native Modules
+----------------------
+
+You can ``require()`` native Node modules (such as ``fs``) in your
```

Comment:
[optional suggestion] Link ``fs`` to https://nodejs.org/api/fs.html#fs_file_system? I like the seealso to explain the specific command, but I think it's a little strange without having linked to 'fs' first/ breaks the pattern below of linking to the thing you introduce.

Reply:
Did a bit more standardizing in the examples. Good call.